README.md

```diff
@@ -99,19 +56,17 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
```

Comment:
- [ ] @team kindly remove all the guidelines from all your section, they are meant to guide you when creating the readme and should be removed when it's done